### ISSUE TITLE
Breaking force for equipped items is not strong enough

### ISSUE THREAD

**Issue by ihsoft** [ihsoft/KIS#128]
By default `physics` parts are attached to kerbounaut with joint that can sustain breaking force at maximum 5 (newtons). In Unity 5 this value is too low.

Long term fix: make it configurable. Short term fix: just raise it x10 as it's done for the static parts.

**Comment by enceos** [comment]
Yes! Save them falling kerbals! )

**Comment by enceos** [comment]
I don't think it should be configurable, just high enough not to fall off. Because even if the part detaches from a Kerbal the game still considers them two a single vessel and the camera dances between the Kerbal and the severed part. Plus the part is still in the inventory.

**Comment by ihsoft** [comment]
I'll set it to 50 for 1.2.8. Only because on the other parts I noticed x10 increase gave good result.

Given the normal weight of a regular kerbal (93kg) and Kerbin physics it should be enough to survive at reasonable fall speeds. Though, I'll make it configurable in the future versions. KSP 1.1 is not yet stable.

**Comment by enceos** [comment]
1.1.2 has been released today, looks like there won't be any KSP patches for a month. Squad goes on vacation.

**Comment by ihsoft** [comment]
I think breaking force should be a settings of KISItem. In normal case (e.g. with a wrench) it sounds reasonable to hold a tool with force of 5 newtons. And this will be the default. But if you want to attach a chute or a tether cable then you sure need a stronger attachment.

**Comment by enceos** [comment]
@ihsoft there's one bad issue with the breaking mechanic, as I described above, even if a part breaks off the Kerbal and floats away, it's still residing in the Kerbal's inventory and the camera focus goes crazy if the part was heavy. It is still possible to unequip and equip the part again as if it had a rope attached to it.

**Comment by ihsoft** [comment]
In my tests when chute got broken the camera was fine. How do I reproduce the issue?

**Comment by enceos** [comment]
@ihsoft click this image to see the video:
[![ScreenShot](http://img.youtube.com/vi/JC6iGuX2nfk/0.jpg)](https://www.youtube.com/watch?v=JC6iGuX2nfk)

Reproduction steps: Equip the KIS Container in physic mode, see it fall to the ground and try walking away from it.
